setup.py

```diff
@@ -23,6 +23,8 @@
     install_requires=[
         "attrs>=21.3.0",
         "itemadapter>=0.2.0",
+        "typing-extensions>=4.3.0",
```

Comment:
Would it make sense to install it only in Python versions which don't support TypeGuard?

Reply:
Good call! https://github.com/zytedata/zyte-common-items/pull/11/commits/6690ed8cbc3091f7888c70dc0b4281c32a30a556

Reply:
Nice catch! https://github.com/zytedata/zyte-common-items/pull/11/commits/f5a9088017792cd8cb3041ee5e4d7f9e5e09b208